.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
             matrix:
                 browser: [Chrome, Firefox]
                 python-version: ['3.10']
-                aiida-core-version: [2.1.2, 2.4.3]          # test on the latest and the oldest supported version
+                aiida-core-version: [2.1.2, 2.5.0]          # test on the latest and the oldest supported version
```

Comment:
We don't have that image yet so this was bound to fail.

Reply:
Let's revert back to v2.4.3 and merge this first. I think then we might proceed with releasing the image.

Once this is merged, maybe you could make a alpha pre-release of AWB (2.2.0a0) which we can then use for both testing the image and QeApp (which needs to be updated to specify optimade extra dependency.

Reply:
Releasing the image of bumping aiida-core to 2.5.0 you mean? But the test will fail since awb is not yet support 2.5.0. 

This is a chicken egg problem. I think we should make a release of awb without changing the aiida-core version in notebook CI test. Make the release and then update the image.

Reply:
Sorry for not being clear. I agree we can't release the image just yet, and we therefore need to keep the 2.4.3 version the the notebook tests here.  Here are the steps that are needed afaik:

1. Merge this PR
2. Release a pre-release version of AWB
3. Make QeApp work with the pre-released version. Verify that it works in the axisting aiidalab images.
4. Release new version of AWB, v2.2.0
5. Release new version of QeApp
6. Release the new image with aiida v2.5

Reply:
Thanks for clarification. But I think it will not working for the step 6.

> Release the new image with aiida v2.5

This is impossible if AWB not yet have pydantic v2 supported. 

aiidalab docker stack full stack test:
aiida-core 2.5.0 -> pydantic v2 -> aiidalab-qe (pydantic v2 support) -> AWB (pydantic v2 support)

So if not yet AWB pydantic v2 supported 

For AWB test, it depend on the image with pydantic v2 supported, but that image depend on the this as shown above.

Reply:
But anyway, before step 6, I think it is all doable. Let's get this merged and when proceed with can see the potential problem more clear and able to find the solution.